src/foundation.cs

```diff
@@ -292,51 +300,60 @@ partial interface NSAttributedString : NSCoding, NSMutableCopying, NSSecureCodin
 		// New API in iOS9 with same signature as an older alternative.
 		// We expose only the *new* one for the new platforms as the old
 		// one was moved to `NSDeprecatedKitAdditions (NSAttributedString)`
+		[NoMac]
```

Comment:
Can you remove the `#if MONOMAC` condition at line 297?

Reply:
Not easily, as multiple Constructor APIs nearly but not completely match macOS ones defined below, and I got a bunch of "can't overload based on ref/out" errors. 

I think long term it's a good idea, but this PR got complicated enough already...